### ISSUE TITLE
rpn foreground accuracy is always 0

### ISSUE THREAD

**Issue by chengweige** [zhaoweicai/cascade-rcnn#81]
i trained the alex-9s-600-rpn-base without finetune,here is the snapshort result
![default](https://user-images.githubusercontent.com/6008657/53884756-4413ee00-4057-11e9-824c-9f5a84e19e85.png)
but the rpn foreground accuracy is always 0, is it alright?

**Comment by zhaoweicai** [comment]
It seems not right. Try finetuning first, and compare the difference. I don't think training-from-scratch works if you don't make any change, e.g. hyper-parameters.

**Comment by chengweige** [comment]
Thank you for your reply. I have solved the problem.

**Comment by xiaosibai** [comment]
Hi！The IOU always equals -1 when i use this "res101-9s-600-rfcn-cascade" to train my data ,can you tell me the reason!? And how to get the value of anchors?  The loss ( floating up and down at 0.5)won‘t drop no mater how i try? Those problems are really killing me now, I cann't figure it out, can you help me?![1](https://user-images.githubusercontent.com/19648223/63021427-259e0280-bed3-11e9-92b0-87212fe3779d.jpg)
